modules/react/badge/lib/CountBadge.tsx

```diff
@@ -28,17 +28,18 @@ export interface CountBadgeProps extends CSProps {
   limit?: number;
   /**
    * Sets the variant of the Count Badge
-   *
-   * @default 'default'
    */
-  variant?: 'default' | 'inverse';
+  variant?: 'inverse';
```

Comment:
Removing the `default` variant and pulling those styles into `base`. This makes the CSS easier to write.

Reply:
this is a breaking change and we need to call this out in upgrade guide